python/tk_desktop/desktop_window.py

```diff
@@ -664,13 +664,11 @@ def _handle_close_event(self):
         if has_warned_of_closed_behaviour is False:
             # On macOS the app icon is already visible on the pop-up
             # message, do not show it a second time.
-            # In Qt4, we can't have a custom icon, so don't display
-            # anything. All the stock Qt icons make it look worse.
-            if sgtk.util.is_macos() or QtCore.qVersion()[0] == "4":
+            if sgtk.util.is_macos():
                 icon = self.systray.NoIcon
             else:
                 icon = self._app_icon
-            # Note that on macOS+Qt4, this call will not show anything,
+            # Note that on macOS, this call will not show anything,
```

Comment:
Actually, we might want to get rid of the whole comment if proven that it now works on macOS. 
Can you try it to confirm?